### ISSUE TITLE
External event drop broken on touch devices when upgrade from 3.1.0 to 3.2.0

### ISSUE THREAD

**Issue by Vohidjon** [fullcalendar/fullcalendar#3544]
I recently upgraded to fullcalendar 3.2.0 and it broke external event drop functionality on touch devices (tested on iPad and chrome developer tools device simulator). 
This [plunker](https://plnkr.co/edit/364sMw?p=preview) works with v3.1.0 and [this one](https://plnkr.co/edit/siCUG1?p=preview) with v3.2.0 does not.
Just in case, everything works fine on normal broswers.

**Comment by arshaw** [comment]
thanks, reproduced.

i'm running this page in emulated tablet mode in chrome:
https://run.plnkr.co/7qupUsVjW6YarIn7/

and i'm able to drag the event, but not able to drop it on any of the day cells.

**Comment by Vohidjon** [comment]
I wanted to solve the problem and make a pull request. However, things aren't that simple. 
Anyway, I did some debugging on this issue.
The root of the bug goes to `getEvIsTouch` method which falsely determines `this.isTouch` property on touch devices . `ev.type` actually equals to `'dragstart'`, so returns false. 
```
function getEvIsTouch(ev) {
	return /^touch/.test(ev.type);
}
```
After comparing 3.1.0 and 3.2.0, I found `getEvIsTouch`  method was not altered at all. So that means it used to return `false` before, too and it was working as expected. 
I hope it helps to fix the bug.
